### PR TITLE
DES-642 Table row templates

### PR DESCRIPTION
**Pull request by lostkeys**
Demo using the table clipboard api's to insert templated table rows

### CODE REVIEW

File: tables/table-row-templates.html [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+    set and paste the clipboard contents programatically
+-->
+
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Table row templates</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinymce/5/tinymce.min.js" referrerpolicy="origin"></script>
```
Comment (ashfordneil) [inline tables/table-row-templates.html:18]: is this api key safe to be public?

File: tables/table-row-templates.html [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+                        {
+                            type: 'submit',
+                            name: 'submitButton',
+                            text: 'Insert',
+                            primary: true
+                        }
+                        ],
+                        onSubmit: (dialog) => {
+                            // Get the form data.
+                            var data = dialog.getData();
```
Comment (ashfordneil) [inline tables/table-row-templates.html:162]: should this be const?

File: tables/table-row-templates.html [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+                        <td style="width: 6.640395473742197%; height: 22px;">2d</td>
+                        <td style="width: 11.281157453835794%; height: 22px;"><span class="badge"     style="background-color: #9b59b6;">Ready</span></td>
+                        <td style="width: 20.47793288946486%; height: 22px;"><a href="https://goo.gl/maps/    bpzGckcmqQpKjgFm7">65.793492, -16.620532</a></td>
+                        <td style="width: 20.339181323978078%;">&nbsp;</td>
+                    </tr>
+                    <tr style="height: 22px;">
+                        <td style="width: 14.609547144693355%; height: 22px;">Aurora</td>
+                        <td style="width: 6.640395473742197%; height: 22px;">44d</td>
+                        <td style="width: 11.281157453835794%; height: 22px;"><span class="badge" style="background-            color: #007bff;">Running</span></td>
+                        <td style="width: 20.47793288946486%; height: 22px;"><a href="https://goo.gl/maps/            cukQ1qtvMnWwtsN28">64.900901, -22.672074</a></td>
```
Comment (ashfordneil) [inline tables/table-row-templates.html:245]:
```suggestion
                        <td style="width: 20.47793288946486%; height: 22px;"><a href="https://goo.gl/maps/cukQ1qtvMnWwtsN28">64.900901, -22.672074</a></td>
```

**Review by ashfordneil — APPROVED** [review]

File: index.html [hunk 4]
```diff
@@ -351,6 +343,29 @@ <h3><a href="tokens/span-based-token.html">Span-based tokens</a></h3>
...
+                <h3><a href="tables/customizing-the-table-toolbar.html">An alternate table context toolbar</a></h3>
+                <p>TinyMCE 5.4 introduced a couple of new table APIs we can use to construct an alternative table context toolbar. In this demo showcases a table header row/column toggle, a menu for choosing cell background color menu among other things.</p>
+                <span class="tag">Intermediate</span>
+                <span class="tag tag-buttons-and-menus">Buttons & Menus</span>
+                <span class="tag tag-plugin">Plugin:Tables</span>
+            </article>
+            
+            <article>
+                <h3><a href="tables/table-row-templates.html">Table row templates</a></h3>
+                <p>t.b.d</p>
```
[the remark this answers is not in the document]
Comment (lostkeys) [reply index.html:363]: No, God I'm sloppy 😦

File: tables/table-row-templates.html (hunk 1 above)
Comment (lostkeys) [reply tables/table-row-templates.html:18]: Yes, that's the API key we use for demos, codepens etc.

File: tables/table-row-templates.html (hunk 3 above)
Comment (lostkeys) [reply tables/table-row-templates.html:245]: Weird, where did those spaces come from?

**Review by SimonFc — APPROVED** [review]

**Pull request merged**